example/index.tsx

```diff
@@ -26,11 +26,11 @@ const App = () => {
         <MSWToolbar
           worker={worker}
           apiUrl="http://www.example.com"
-          isEnabled={true}
+          isEnabled
           actions={
             <HStack spacing={2}>
               <Button onClick={() => alert('Hooray')}>Custom Action</Button>
-              <Button onClick={() => alert('Yadiddit')}>Another</Button>
+              <Button onClick={() => alert('Yadonediddit')}>Another</Button>
```

Comment:
🤣

Reply:
@Jarvis1010 Any thoughts here?

Reply:
I think you are missing about 3 exclamation points after your Yadonediddit.  Also have you considered wrapping `alert` in a `dagnabit` function?